packages/kernel/vitest.config.ts

```diff
@@ -17,7 +17,7 @@ export default defineConfig({
       ),
       "streamlit.whl": path.resolve(
         __dirname,
-        "./py/streamlit/lib/dist/streamlit-1.41.0-cp312-none-any.whl",
+        `./py/streamlit/lib/dist/streamlit-${process.env.STREAMLIT_VERSION || "1.41.0"}-cp312-none-any.whl`,
```

Comment:
What provides the `STREAMLIT_VERSION` environment variable?
Looks like it's now working just because of the fallback value. Delete the fallback value for the "fail early" principle.